.github/bin/format-package-json.mjs

```diff
@@ -199,7 +200,7 @@ const formatted = {};
 Object.assign(formatted, packageJSONInfo);
 
 if (process.env.GITHUB_ACTIONS && JSON.stringify(formatted, null, '\t') !== packageJSONInfoCopy) {
-	console.error('package.json has an incorrect field order. Run "npm run lint" to resolve.')
+	console.error(`::error file=${path.relative(process.env.GITHUB_WORKSPACE, path.resolve('./package.json'))},line=1,col=1::package.json has an incorrect field order. Run "npm run lint" to resolve.`);
```

Comment:
Didn't want to include the entire formatting logic here as it is a pretty static message.